rotkehlchen/assets/converters.py

```diff
@@ -880,7 +920,13 @@ def asset_from_gemini(symbol: str) -> Asset:
     """
     if not isinstance(symbol, str):
         raise DeserializationError(f'Got non-string type {type(symbol)} for gemini asset')
-    return symbol_to_asset_or_token(symbol)
+
+    if symbol in UNSUPPORTED_GEMINI_ASSETS:
+        raise UnsupportedAsset(symbol)
+
+    name = GEMINI_TO_WORLD.get(symbol, symbol)
+
+    return symbol_to_asset_or_token(name)
```

Comment:
Look more carefully in my comments. I had also asked for removal of whitespace.

```suggestion
    name = GEMINI_TO_WORLD.get(symbol, symbol)
    return symbol_to_asset_or_token(name)
```

Don't do it now though, no need to run all tests for this again.